main/CMakeLists.txt

```diff
@@ -87,8 +87,8 @@ if (BUILD_UI_MU4)
         context
         shortcuts
         workspace
-        audio_engine
-        audio_midi
+        mu4_audio
```

Comment:
Is there any naming conflict? Like another audio module or something?
I mean why do we need prefix "mu4"?

Reply:
Yes, your guess is correct, there is a name conflict.
Therefore, the prefix "mu4" is temporarily added

Reply:
I'd like to suggest to use the prefix "framework" or "fw".